### ISSUE TITLE
Right-button menu for Scene-file nodes has a bigger offset than normal

### ISSUE THREAD

**Issue by henriquelalves** [godotengine/godot#37178]
**Godot version:** 3.2.1 (stable)

**OS/device including version:** ArchLinux

**Issue description:** When clicking with the right-button on a node added to the scene via a Scene file, the inspector menu that appears will have a weird offset (as shown in the images). This is probably caused by the toggle options "Editable Children" and "Load as Placeholder", as they show more distant to the option label than the rest of the icons.

This is a (very) minor UI bug and I couldn't found something similar happening somewhere else in Godot.

Normal Menu:
![normal](https://user-images.githubusercontent.com/7095429/77133195-fd661b80-6a40-11ea-8772-722b7e8eeb5e.png)

Weird Offset Menu:
![bugged](https://user-images.githubusercontent.com/7095429/77133193-fccd8500-6a40-11ea-9481-9620e7a6d94a.png)

**Comment by Calinou** [comment]
I don't think this is actually bug, just a limitation of the PopupMenu system which doesn't allow an icon to have both an icon and checkbox at the same place.

**Comment by henriquelalves** [comment]
Damn. Isn't it better to change the issue tags from 'bug' to 'enchancement', then? Just to leave the issue so someone can tackle it in the future, if the PopupMenu is ever refactored.

**Comment by dankan1890** [comment]
Just decide which behavior to use:
1) checkbox and icon in the same place (with the limitation that the item cannot have both):

![libreoffice_style](https://user-images.githubusercontent.com/7107228/77237107-7ee6b680-6bc5-11ea-9408-259836b8a47e.png)

2) always leave space for the checkbox:

![edge_style](https://user-images.githubusercontent.com/7107228/77237109-86a65b00-6bc5-11ea-814a-d2353d528011.png)

**Comment by mason-bially** [comment]
As they don't currently have icons I think the "use checkbox as icon" is probably the more elegant UI solution but likely the more complicated code solution.

**Comment by EricEzaM** [comment]
@Calinou @akien-mga @groud Any executive decision on this on which way to go for this issue? I am fine with current implementation tbh.

**Comment by groud** [comment]
Well, if we could make it so that we switch between the 2 behaviors (as proposed by @dankan1890) automatically, it would be perfect. The idea would be to check beforehand if there is at least one menu entry with both a checkbox and an icon.

**Comment by EricEzaM** [comment]
I am not convinced that merging them into one column is a good idea. I think it would lose some visual clarity. Is it an icon or a checkbox? Observe: where are the checkboxes below? It takes time to find them.
![image](https://user-images.githubusercontent.com/41730826/91620613-c1fa5080-e9d3-11ea-8add-40389d524233.png)

**Comment by groud** [comment]
> I am not convinced that merging them into one column is a good idea. I think it would lose some visual clarity. Is it an icon or a checkbox? Observe: where are the checkboxes below? It takes time to find them.

Not that much if we make the checkbox have an "highlight" using a contour or something like when when hovering the option.
This could likely be done easily but requires a dedicated PR.
